crysthack_day1_GitHub_workflow_exercise/src/main.py

```diff
@@ -1,6 +1,26 @@
+import glob
+import random
+
+
+def read_names():
```

Comment:
to make this more reusable I would pass in the file-path.  In our group we have pretty much moved over all our path handling to `pathlib.Path` and pass things around as Path objects.

Reply:
Excellent! I was wondering what would be a cleaner way to keep track of paths